src/measure_theory/function/uniform_integrable.lean

```diff
@@ -213,7 +213,7 @@ begin
     rwa compl_eq_univ_diff },
   { filter_upwards [hfg],
     intros,
```

Comment:
This one might be an accidental omission?

Reply:
I was not sure about that one either, but it broke when I tried to merge it with `filter_upwards` so I just left it as it was.

Reply:
Alright, after downloading cached olean files I was able to actually open the file on VS Code and change it while looking at the goal state 👍🏼